app/common/state/aboutPreferencesState.js

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const assert = require('assert')
+
+// utils
+const {makeImmutable, isMap} = require('../../common/state/immutableUtil')
+
+const validateState = (state) => {
+  state = makeImmutable(state)
+  assert.ok(isMap(state), 'state must be an Immutable.Map')
+  assert.ok(isMap(state.get('ledger')), 'state must contain an Immutable.Map of ledger')
```

Comment:
we should check here if `state.getIn(['about', 'preferences'])` instead of `state.get('ledger')`